_posts/dev/2014-08-02-kura-dio-apis.md

```diff
@@ -128,6 +130,205 @@ button.setInputListener(new PinListener(){
 		}
 });
 
+```
+### Accessing a GPIO Pin with GPIOService for a switch a Led. 
+
+Inside the **jdk.dio.properties** file:
+
+```
+...
+6 = deviceType: gpio.GPIOPin, pinNumber:6, name:GPIO06
+...
+```
+
+Then in the code:
+
+```java
+
+private GPIOService myservice;
+
+KuraGPIOPin pin = this.myservice.getPinByTerminal(6);
+
+// to allow pin configuration use
+//pin = this.myservice.getPinByTerminal(this.options.isConfigPin());
+
+``` 
+In a **component.xml** enter a new service. Select a **Service** and in a **Referenced Service** open **Add** for add  the GPIOService. Select the **GPIOService** and use edit button for set in the bind field to **bindGPIOService** and set the unbind filed to **unbindGPIOService**.
```

Comment:
It's probably better to associate this description with a screenshot. Or provide the final xml